### PR TITLE
Add color-adjust property example

### PR DESCRIPTION
**Pull request by a2sheppy**
Adds color-adjust interactive example.

### CODE REVIEW

File: live-examples/css-examples/color/color-adjust.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#example-element {
+    padding: 1em;
+    font-size: 1.5em;
```
Comment (schalkneethling) [inline live-examples/css-examples/color/color-adjust.css]: Super Nit: Code style: Prefer `rem` over `em` for font sizes.

File: live-examples/css-examples/color/color-adjust.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    background-color: black;
+    background-image: linear-gradient(rgba(0, 0, 180, 0.5), rgba(70, 140, 220, 0.5));
```
Comment (schalkneethling) [inline live-examples/css-examples/color/color-adjust.css]: Nit: Code style: Omit `0` from fractional values

**Comment by a2sheppy** [comment]
Remove the leading "0" from fractional numbers? That's... strange. Leaving out the leading 0 makes the number harder and violates every style guide I've ever seen.

**Comment by a2sheppy** [comment]
It's still weird; I've honestly never seen such a thing. I'll make the change, but I think it badly damages legibility of the code.

**Comment by chrisdavidmills** [comment]
I've seen both in fairly equal measure, FWIW. I think I prefer "with leading 0s", for similar reasons to those sheppy brings up, but I can live with it.

**Comment by schalkneethling** [comment]
@a2sheppy @chrisdavidmills Yeah, this is one of those points where there is often contention ;) As we have followed this guideline for the rest of this codebase, and have also been following it on Kuma since before my time, I felt that we should stick with it.

**Pull request merged**